superset-frontend/src/dashboard/actions/hydrate.js

```diff
@@ -79,7 +113,7 @@ export default function getInitialState(bootstrapData) {
   }
 
   // dashboard layout
-  const { position_json: positionJson } = dashboard;
+  const positionJson = JSON.parse(dashboardData.position_json);
```

Comment:
var name nit: once parsed, it's just the positionObj or positionValue or something, right?